File: go/orm/sample/migrations/V0004.go
package migrations

import (
	"github.com/kosmassp/SimpleOrm/go/orm"
	"github.com/kosmassp/SimpleOrm/go/orm/sample/migrations/Table/Role"
)

type V0004 struct{}

func (V0004) Compose(version *orm.VersionBuilder) {
	version.Apply(role.V0004_RenameNameToRoleName{})
}

File: php/samples/SimpleOrm.Sample/src/Migrations/V0004.php
<?php

declare(strict_types=1);

namespace SimpleOrm\Sample\Migrations;

use SimpleOrm\Migrations\MigrationVersion;
use SimpleOrm\Migrations\VersionBuilder;
use SimpleOrm\Sample\Migrations\Table\Role\V0004_RenameNameToRoleName;

final class V0004 extends MigrationVersion
{
    public function compose(VersionBuilder $version): void
    {
        $version
            ->apply(V0004_RenameNameToRoleName::class);
    }
}
